package.json

```diff
@@ -0,0 +1,109 @@
+{
+    "name": "@eyeseetea/feedback-component",
+    "description": "Feedback component for DHIS2 apps",
```

Comment:
It could be probably be used on any React app, not only DHIS2.

Reply:
"Form for receiving feedback from user" would be fine?